docker/entrypoint_check.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh -l
+
+echo '\nGetting the code...\n'
+git clone --quiet https://$INPUT_TOKEN@github.com/$GITHUB_REPOSITORY check
+cd check
+git checkout $BRANCH_SOURCE
+rm .Rprofile
+
+echo '\nSession info\n'
+Rscript -e 'sessioninfo::session_info()'
+
+echo '\nUpdate version number...\n'
+git config --global user.email "info@inbo.be"
+git config --global user.name "INBO"
+UPDATED=$(Rscript -e 'protocolhelper::update_version_number("'$PROTOCOL_CODE'")')
+echo 'output updated:' $UPDATED
+if [ "$UPDATED" = "[1] TRUE" ]; then
+  # remake the last commit without deletion of .Rprofile
+  MESSAGE=$(git log -1 --pretty=%B)
+  git reset --soft HEAD^
+  git reset HEAD .Rprofile
+  git commit --message="$MESSAGE"
+
+  git push -f
```

Comment:
pushing a new commit will trigger another round of checks.

Reply:
Eh, I'm not sure about that.  I thought I'm using a token to authenticate, and [here](https://docs.github.com/en/actions/using-workflows/triggering-a-workflow#:~:text=When%20you%20use%20the%20repository%27s%20GITHUB_TOKEN%20to%20perform%20tasks%2C%20events%20triggered%20by%20the%20GITHUB_TOKEN%20will%20not%20create%20a%20new%20workflow%20run.%20This%20prevents%20you%20from%20accidentally%20creating%20recursive%20workflow%20runs) Github states a new check is not triggered in case a GITHUB_TOKEN is used (to avoid recursive triggers).  Or is this case different since I configure the user name?